### ISSUE TITLE
crossposted: armv7l version of sbt won't build simple project

### ISSUE THREAD

**Issue by apple-corps** [chromebrew/chromebrew#6071]
https://github.com/sbt/sbt/issues/6622

**Comment by drocsid** [comment]
https://stackoverflow.com/a/45536678

**Comment by drocsid** [comment]
@satmandu @uberhacker how might i install java 11 so I can test against J11? I saw a reference to Java 16 somewhere so assume the jdk8.rb might install jdk 11.

**Comment by satmandu** [comment]
@uberhacker Looks like a java issue?

**Comment by satmandu** [comment]
@drocsid Try `crew install jdk11` .

**Comment by drocsid** [comment]
doh! It doesn't look like Oracle supports armv7l releases of JDK11. However there are other sources of armv7l JDK11 releases including adoptopenJDK and bellsoft. Maybe I will try one of them. But I'm expecting the Java 8 armv7l release should work.

**Comment by uberhacker** [comment]
Yeah, it's a bit annoying that so many projects are dropping arm support.  I'm guessing it's just laziness.

**Comment by drocsid** [comment]
to be fair there's 64bit support for aarch 64 via Oracle JDK 11. Looks like they dropped the 32bit support.
